Parsers/ASimWebSession/Parsers/ASimWebSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,452 @@
+Parser:

```

Comment:
@jaimeesc Like you did for Network parser. Can you please add SonicWall EventVendor and EventProduct values in tester.csv for WebSession schema as well?

Reply:
I did that on the Network Session parser PR, but the file diff was really strange. I rolled back the file to see if that helps. I'm not sure what is causing the diff to appear as if I'm making large changes.

Reply:
Quick question on this one. Do I need to include this change in this PR even though it has already been added in another PR?

Reply:
@jaimeesc 
For every schema we need to add supported EventVendor and EventProduct. If in the other PR, you added EventVendor and EventProduct for Network and WebSession schema, then not needed else please update in this PR.

Reply:
Excellent, thank you. Already submitted in the other PR so we should be good.